### PR TITLE
[file_selector_web] update documentation #76067

### PR DESCRIPTION
**Pull request by bharat-biradar**
This PR updates the dependency versions in [file_selector_web readme.md](https://github.com/flutter/plugins/blob/master/packages/file_selector/file_selector_web/README.md)

Fixes [flutter/flutter#76067](https://github.com/flutter/flutter/issues/76067)

## Pre-launch Checklist

- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: packages/file_selector/file_selector_web/README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ This is what the above means to your `pubspec.yaml`:
 ...
 dependencies:
   ...
-  file_picker: ^0.7.0
-  file_picker_web: ^0.7.0
+  file_picker: ^0.7.0+2
+  file_picker_web: ^0.7.0+1
```
Comment (stuartmorgan) [inline packages/file_selector/file_selector_web/README.md:22]: The problem is the name being wrong, not the exact versions, so this doesn't actually fix the issue.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/file_selector/file_selector_web/README.md (hunk 1 above)
Comment (ditman) [inline packages/file_selector/file_selector_web/README.md:22]:
```suggestion
  file_selector: ^0.7.0
  file_selector_web: ^0.7.0
```

File: packages/file_selector/file_selector_web/README.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 ```
 
 ### Use the plugin
-Once you have the `file_picker_web` dependency in your pubspec, you should
-be able to use `package:file_picker` as normal.
+Once you have the `file_selector_web` dependency in your pubspec, you should
+be able to use `package:file_selector` as normal.
 
-[1]: ../file_picker/file_picker
+[1]: ../file_selector/file_selector
```
Comment (ditman) [inline packages/file_selector/file_selector_web/README.md:30]:
```suggestion
[1]: https://pub.dev/packages/file_selector
```

File: packages/file_selector/file_selector_web/README.md (hunk 1 above)
Comment (ditman) [reply packages/file_selector/file_selector_web/README.md]: Updated the branch with the correct fix, PTAL :)

**Review by stuartmorgan — APPROVED** [review]
LGTM, thanks @ditman for cleaning it up!

**Comment by ditman** [comment]
Merging as soon as CI goes green! Thanks for the contribution @bharat-biradar!

**Pull request merged**